crypto/include/internal/sparse_array.h

```diff
@@ -20,40 +20,40 @@ extern "C" {
 # define DEFINE_SPARSE_ARRAY_OF(type) \
     SPARSE_ARRAY_OF(type); \
     static ossl_inline SPARSE_ARRAY_OF(type) * \
-        sa_##type##_new(void) \
+        ossl_sa_##type##_new(void) \
     { \
         return (SPARSE_ARRAY_OF(type) *)OPENSSL_SA_new(); \
     } \
-    static ossl_inline void sa_##type##_free(SPARSE_ARRAY_OF(type) *sa) \
+    static ossl_inline void ossl_sa_##type##_free(SPARSE_ARRAY_OF(type) *sa) \
     { \
         OPENSSL_SA_free((OPENSSL_SA *)sa); \
     } \
-    static ossl_inline void sa_##type##_free_leaves(SPARSE_ARRAY_OF(type) *sa) \
+    static ossl_inline void ossl_sa_##type##_free_leaves(SPARSE_ARRAY_OF(type) *sa) \
     { \
         OPENSSL_SA_free_leaves((OPENSSL_SA *)sa); \
     } \
-    static ossl_inline size_t sa_##type##_num(const SPARSE_ARRAY_OF(type) *sa) \
+    static ossl_inline size_t ossl_sa_##type##_num(const SPARSE_ARRAY_OF(type) *sa) \
     { \
         return OPENSSL_SA_num((OPENSSL_SA *)sa); \
     } \
-    static ossl_inline void sa_##type##_doall(const SPARSE_ARRAY_OF(type) *sa, \
+    static ossl_inline void ossl_sa_##type##_doall(const SPARSE_ARRAY_OF(type) *sa, \
                                               void (*leaf)(type *)) \
```

Comment:
Indentation!

Reply:
Drat, missed it :(

Reply:
Some of the macro expansion lines at >80 characters long too.  I don't know how to address this.  Even breaking at natural places results in lines that are too long.

Reply:
@levitte is it allowed by current style guidelines to break even the first argument of a function definition and put it on the following line with less whitespace than the opening parenthesis in this case?

Reply:
One possibility is to break between the return type and the function name, i.e. this:

``` C
    static ossl_inline void \
    ossl_sa_##type##_doall(const SPARSE_ARRAY_OF(type) *sa, \
```

We might want to double check what the code style has to say about that, but I also think that we need to consider being flexible in cases like this.

Another possibility that I've used (and absolutely hated to do) is to break up the parameter spec, like this:

``` C
    static ossl_inline void ossl_sa_##type##_doall(const \
                                                   SPARSE_ARRAY_OF(type) \
                                                   *sa, \
```

Quite frankly, I find that incredibly ugly and would much prefer breaking between return type and function name (i.e. my first suggestion).